design/FY2021/NFP-Waterside-Economizer.md

```diff
@@ -0,0 +1,795 @@
+         \key Coil:Cooling:DX:TwoSpeed
+         \key Coil:Cooling:DX:TwoStageWithHumidityControlMode
+         \key Coil:Cooling:DX:VariableSpeed
+         \key Coil:Cooling:DX:SingleSpeed:ThermalStorage
+
+    CoilSystem:Heating:DX,
+    A3,  \field Heating Coil Object Type
+         \key Coil:Heating:DX:SingleSpeed
+         \key Coil:Heating:DX:VariableSpeed
+
+    CoilSystem:Cooling:Water:HeatExchangerAssisted,
+    A2 , \field Heat Exchanger Object Type
+         \key HeatExchanger:AirToAir:FlatPlate
+         \key HeatExchanger:AirToAir:SensibleAndLatent
+    A4 , \field Cooling Coil Object Type
+         \key Coil:Cooling:Water
+         \key Coil:Cooling:Water:DetailedGeometry
+
+    CoilSystem:Cooling:DX:HeatExchangerAssisted,
+    A2 , \field Heat Exchanger Object Type
+         \key HeatExchanger:AirToAir:FlatPlate
+         \key HeatExchanger:AirToAir:SensibleAndLatent
+         \key HeatExchanger:Desiccant:BalancedFlow
+    A4 , \field Cooling Coil Object Type
+         \key Coil:Cooling:DX:SingleSpeed
+         \key Coil:Cooling:DX:VariableSpeed
+
+It may be time to begin consolodating these exising wrappers as done with the Coil:Cooling:DX object. The name would simply be CoilSystem and contain any type of coil with an optional heat exchangera available for cooling coils. This big of a change may be too big for this task, however, keeping this in mind for this new object is highly desireable.
+
+**Instead of incorporating a waterside economizer coil object into a packaged system, a generic method of providing free cooling is discussed here. The new object would be configured as a condenser plant demand side component and placed upstream of any existing coil types. A new wrapper will be developed to include the water coils. This wrapper could be specific to a new waterside economizer coil object or be more generic to allow branch water coils to be simulated without using the Controller:WaterCoil object. The more generic method is described here.**
+
+**Note**: This new object may also be used on the demand side of a cooling plant to support existing coil types.
+
+## Input Description ##
+
+
+    CoilSystem:Cooling:Water
+        \memo Virtual container component that consists of a water cooling coil
+        \memo and its associated controls. This control object supports the
+        \memo available water coil types and may be placed directly on an
+        \memo air loop branch or in an outdoor air equipment list.
+        \min-fields 6
+    A1, \field Name
+        \required-field
+        \reference WaterCoilSystemName
+        \type alpha
+        \reference-class-name validBranchEquipmentTypes
+        \reference validBranchEquipmentNames
+        \reference-class-name validOASysEquipmentTypes
+        \reference validOASysEquipmentNames
+    A2, \field Cooling Coil System Inlet Node Name
+        \required-field
+        \type node
+    A3, \field Cooling Coil System Outlet Node Name
```

Comment:
I would prefer Air Inlet Node Name, and Outlet.
Or we could follow the model of CoilSystem:Heating:DX which doesn't even have node names. I assume it mines those from the child coil.

Reply:
OK, I will do either of the two suggestions.